frontend/src/components/InlineMessage/InlineMessage.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story } from '@storybook/react'
+
+import { InlineMessage, InlineMessageProps } from './InlineMessage'
+
+export default {
+  title: 'Components/InlineMessage',
+  component: InlineMessage,
+  argTypes: {
+    variant: {
+      options: ['info', 'warning', 'error'],
```

Comment:
does it not autoconvert the argTypes with the exported `InlineMessageVariant` type? (may need to restart storybook so the builder rebuilds the app)

Reply:
it does actually, made a typing mistake, my bad. corrected